packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.controller.js

```diff
@@ -284,4 +286,48 @@ export default class DedicatedServerDashboard {
       value.unit === 'bps' ? (value.y / 1024).toFixed(2) : value.y,
     );
   }
+
+  loadAdvices() {
+    const { commercialRange } = this.server;
+    if (commercialRange && commercialRange.startsWith('ADVANCE')) {
+      const existingBandwidth = get(
+        this,
+        'specifications.bandwidth.OvhToInternet.value',
+      );
+      return this.getBandwidth(existingBandwidth).then((plans) => {
+        const bandwidthGuaranteedPlans = find(
+          plans,
+          (plan) =>
+            plan.planCode === 'bandwidth-guaranteed-1000-ultimate-2019v1' ||
+            plan.planCode === 'bandwidth-guaranteed-1000-ultimate-2019v1-us',
+        );
+        if (bandwidthGuaranteedPlans) {
+          return this.getAdvices();
+        }
+        return null;
+      });
+    }
+    return null;
+  }
+
+  getAdvices() {
+    return [
+      {
+        localizedName: this.$translate.instant(
+          'server_advices_dedicated_advice1',
+        ),
+        href: this.orderPublicBandwidthLink,
+        tag:
+          'cross_sell::dedicated::bare_metal_advanced_without_guaranteed_bw::public_bandwidth_1gbps_unmetered_and_guaranteed',
+      },
+    ];
+  }
+
+  getBandwidth(existingBandwidth) {
+    return this.Server.getBareMetalPublicBandwidthOptions(
+      this.server.name,
+    ).then((plans) =>
+      this.Server.getValidBandwidthPlans(plans, existingBandwidth),
+    );
```

Comment:
No need to handle error ?

Reply:
no, if there is any error we don't show up-sell cross-sell devices